projects/github-actions/repo-gardening/src/tasks/gather-support-references/index.js

```diff
@@ -76,32 +68,31 @@ async function getIssueReferences( octokit, owner, repo, number ) {
 
 	// That array can still include duplicates, or references that are not formatted quite properly.
 	// Let's build a final array with unique and correct support IDs.
-	const correctedSupportIds = [];
+	const correctedSupportIds = new Set();
 	supportIds.map( supportId => {
 		// xxx-zen and xxx-hc are the preferred formats for tickets and chats.
 		// xxx-zd and xxx-chat, as well as uppercase versions, are considered as alternate versions.
 		const wrongId = supportId.match( /([0-9]*)-(zd|chat)/i );
 		if ( wrongId ) {
 			const correctedId = `${ wrongId[ 1 ] }-${ wrongId[ 2 ] === 'zd' ? 'zen' : 'hc' }`;
```

Comment:
Should we take the possibility of "ZD" into account?
```suggestion
			const correctedId = `${ wrongId[ 1 ] }-${ wrongId[ 2 ].toLowerCase() === 'zd' ? 'zen' : 'hc' }`;
```